maxString.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"unicode/utf8"
+)
+
+func max(arr []string) string {
+	if arr == nil || len(arr) == 0 {
```

Comment:
```suggestion
	if  len(arr) == 0 {
```

That should be enough, because nil slice has zero length.